packages/amazonq/src/inlineChat/output/computeDiff.ts

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { type LinesOptions, diffLines, Change } from 'diff'
+import * as vscode from 'vscode'
+import { InlineTask, TextDiff } from '../controller/inlineTask'
+
+export function computeDiff(response: string, inlineTask: InlineTask, isPartialDiff: boolean): TextDiff[] | undefined {
```

Comment:
@ivikash is there diff-related code on master or some other branch that might overlap with this module?

Reply:
The existing implementation generates a diff by
- Creating a temporary clone of the file in `tempDir`
- Overwriting the selected Range with the new code.

Trying to reuse the same logic, here will make it more complicated instead.